I adjusted the noblemin and noblefirst features to be "smarter".

ToxicDonut95 opened the pull request:
Since the recent meta on Tribalwars has shifted towards noble spam I adjusted a "bug" with the nobleMin and nobleFirst button. I just made it so Sangu would look at world data to see how much troops are actually required too send your nobles instead of a committed value.

Laoujin commented:
Oh, very nice! I'll try to release this this weekend. Should I forget, ping me!

Laoujin commented:
It has been deployed.
Should become available in the Chrome web store in a few hours/days

ToxicDonut95 commented:
Ye well, I found a bug in my changes which I just fixed. 😝

I’ll probably upload the fix later tonight, sorry!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Wouter Van ***@***.***>
Sent: zondag 11 april 2021 15:19
To: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Subject: Re: [SanguPackage/Script] I adjusted the noblemin and noblefirst features to be "smarter". (#23)


It has been deployed.
Should become available in the Chrome web store in a few hours/days

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/SanguPackage/Script/pull/23#issuecomment-817306872>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATSBGHG5VW36X7EYH4W522LTIGOUPANCNFSM42Q2PNHQ>.
